tests/bootstrap.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*
- * Copyright 2014 Google Inc.
+ * Copyright 2015 Google Inc.
```

Comment:
Oh! Good to know!

Well this was actually written in `2015` and I just forgot to update it when I first committed it.